### ISSUE TITLE
IsRounded = false doesn't work.

### ISSUE THREAD

**Issue by Sebastien-Charton** [stsrki/Blazorise#337]
When I add IsRounded = false to my code the elements stay rounded.

```
@code {
    Theme theme = new Theme
    {
        IsRounded = false,
        ColorOptions = new ThemeColorOptions
        {
            Primary = "#df691a",
            Success = "#5cb85c",
            Info = "#5bc0de",
            Warning = "#f0ad4e",
            Danger = "#d9534f",
            Secondary = "#4e5d6c"

        },

        BackgroundOptions = new ThemeBackgroundOptions
        {
            Dark = "#252a34"
        }
    };
}

```

**Comment by stsrki** [comment]
I have just tried and it works. Maybe you forgot to add `<ThemeProvider>`? It should be something like this in the `App.razor` file:

```cshtml
<Blazorise.ThemeProvider Theme="@theme">
    <Router AppAssembly="typeof(App).Assembly">
        <Found Context="routeData">
            <RouteView RouteData="routeData" DefaultLayout="typeof(MainLayout)" />
        </Found>
        <NotFound>
            <p>Sorry, there's nothing at this address.</p>
        </NotFound>
    </Router>
</Blazorise.ThemeProvider>
```

**Comment by Sebastien-Charton** [comment]
Here is my original code. The others settings work well except this one.

```
<Blazorise.ThemeProvider Theme="@theme">
    <Router AppAssembly="@typeof(Program).Assembly">

        <Found Context="routeData">
            <RouteView RouteData="@routeData" DefaultLayout="@typeof(MainContainer)" />
        </Found>
        <NotFound>
            <p>Sorry, there's nothing at this address.</p>
        </NotFound>
    </Router>
</Blazorise.ThemeProvider>
```

**Comment by stsrki** [comment]
It seems it's a bug with theme generator. You need to set other options to make it work

```diff
@code {
    Theme theme = new Theme
    {
        IsRounded = false,
        ColorOptions = new ThemeColorOptions
        {
            Primary = "#df691a",
            Success = "#5cb85c",
            Info = "#5bc0de",
            Warning = "#f0ad4e",
            Danger = "#d9534f",
            Secondary = "#4e5d6c"

        },

        BackgroundOptions = new ThemeBackgroundOptions
        {
            Dark = "#252a34"
        },
+       ButtonOptions = new ThemeButtonOptions { },
+       DropdownOptions = new ThemeDropdownOptions { },
+       InputOptions = new ThemeInputOptions { },
+       CardOptions = new ThemeCardOptions { },
+       ModalOptions = new ThemeModalOptions { },
+       TabsOptions = new ThemeTabsOptions { },
+       ProgressOptions = new ThemeProgressOptions { },
+       AlertOptions = new ThemeAlertOptions { },
+       BreadcrumbOptions = new ThemeBreadcrumbOptions { },
+       BadgeOptions = new ThemeBadgeOptions { },
+       PaginationOptions = new ThemePaginationOptions { },
    };
}
```

I will fix it in v0.8.8

**Comment by Sebastien-Charton** [comment]
Yes it's working with your solution ! Thanks for the help and have a good day !
